learners/setup.md

```diff
@@ -2,53 +2,158 @@
+    - vantage6-algorithm-tools
 
-### Details
+### Docker
 
-Setup for different systems can be presented in dropdown menus via a `spoiler`
-tag. They will join to this discussion block, so you can give a general overview
-of the software used in this lesson here and fill out the individual operating
-systems (and potentially add more, e.g. online setup) in the solutions blocks.
+We recommend installing [Docker Desktop](https://www.docker.com/products/docker-desktop/) on your system, which is the easiest way to install Docker.
 
-:::::::::::::::::::::::::::::::::::::::::::::::::::
+You can follow the official instructions to install Docker Desktop on [MacOS](https://docs.docker.com/desktop/install/mac-install/), [Windows](https://docs.docker.com/desktop/install/windows-install/), or [Linux](https://docs.docker.com/desktop/install/linux-install/).
 
-:::::::::::::::: spoiler
+::: callout
+#### Linux users
+If you have Docker Engine installed on your Linux system and want to use it instead of Docker Desktop, you must make sure that it can run Docker containers without `sudo`. For that, please check the [guide](https://docs.docker.com/engine/install/linux-postinstall/) for more information.
+:::
 
-### Windows
+After installing the Docker Desktop, open the application, and then verify the installation by running the command in your terminal:
 
-Use PuTTY
+```bash
+docker version
+```
 
-::::::::::::::::::::::::
+```Output
+Client:
+ Version:           26.1.4
+ API version:       1.45
+ Go version:        go1.21.11
+ Git commit:        5650f9b
+ Built:             Wed Jun  5 11:26:02 2024
+ OS/Arch:           darwin/arm64
+ Context:           desktop-linux
 
-:::::::::::::::: spoiler
+Server: Docker Desktop 4.31.0 (153195)
+ Engine:
+  Version:          26.1.4
+  API version:      1.45 (minimum version 1.24)
+  Go version:       go1.21.11
+  Git commit:       de5c9cf
+  Built:            Wed Jun  5 11:29:12 2024
+  OS/Arch:          linux/arm64
+  Experimental:     false
+ containerd:
+  Version:          1.6.33
+  GitCommit:        d2d58213f83a351ca8f528a95fbd145f5654e957
+ runc:
+  Version:          1.1.12
+  GitCommit:        v1.1.12-0-g51d5e94
+ docker-init:
+  Version:          0.19.0
+  GitCommit:        de40ad0
+```
 
-### MacOS
+The above output shows a successful installation and will vary based on your system. Make sure you see versions for both `Client` and `Server`.
 
-Use Terminal.app
+If you cannot see the `Server` version, you have to open the Docker Desktop application first, then run the command again.
 
-::::::::::::::::::::::::
+### Conda
 
+There are several ways to install Conda on your system. For this lesson, we recommend using Miniconda installer.
 
-:::::::::::::::: spoiler
+Follow the [official instructions](https://docs.anaconda.com/free/miniconda/#quick-command-line-install) to install Miniconda on Windows, MacOS, or Linux.
 
-### Linux
+Open a **new** terminal window and verify the installation by running the command:
```

Comment:
Maybe clarify this even further by saying "After completing the miniconda installation, open a new terminal..."